### PR TITLE
[Feature] add variable request template

### PR DESCRIPTION
**Pull request by wyusuf068**

### CODE REVIEW

**Comment by wyusuf068** [comment]
Similar to issue template, set up a template for variable requests

**Comment by wyusuf068** [comment]
> Add section describing how to calculate the variable for derived variables?

How about asking them to attach a .R file that provides documentation and how the variable is derived?

File: .github/ISSUE_TEMPLATE/variable-request.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Variable request
+about: Suggest a variable for cchsflow
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```
Comment (DougManuel) [inline .github/ISSUE_TEMPLATE/variable-request.md]: Hmm. I think we have different levels that we should consider. 
People may just want to make a request and not do too much work. An intro something like:
"We accept requests and PR for new variables. Providing information about your request helps discussion about whether and how to include the variable."

File: .github/ISSUE_TEMPLATE/variable-request.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+For existing variables in the CCHS.
+
+**Derived variables only. What variables are used to create this variable?**
+List the variables in cchsflow used to create this variable
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+
+**Additional instructions for derived variables**
+Please attach an .R file that includes documentation of the derived variable as per roxygen2 standards, along with the code to derive the variable
```
Comment (DougManuel) [inline .github/ISSUE_TEMPLATE/variable-request.md]: "Derived variables use R code for more complex operations and multiple starting variables. Note: cchsflow currently does not include dependancy for derived variables. 
If possible, attach an .R file that includes documentation of the derived variable as per roxygen2 standards, along with the code to derive the variable. Include all starting variables."

**Pull request merged**